### ISSUE TITLE
chatwoot incoming messages improvements

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#2907]

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}
//set the version type
process.env.VERS = "patch"

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci $VERS`);

// types only package
exec('npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN')
exec(`cd ./types-only && npm version $VERS && npm run build && npm publish && cd ..`);
exec(`git commit -a -m 'updated types-only package'`);
exec(`git push --force`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}
//set the version type
process.env.VERS = "patch"

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci $VERS`);

// types only package
exec('npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN')
exec(`cd ./types-only && npm version $VERS && npm run build && npm publish && cd ..`);
exec(`git commit -a -m 'updated types-only package'`);
exec(`git push --force`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.47.1](https://github.com/open-wa/wa-automate-nodejs/compare/4.47.0...4.47.1) (2022-10-11)

- 🗑️  doc changes #2901 [`ddc3600`](https://github.com/open-wa/wa-automate-nodejs/commit/ddc36003890df33af280bbf94063bbb9af1cfb26)
- ➕ added docusaurus to dev deps [`edd8afb`](https://github.com/open-wa/wa-automate-nodejs/commit/edd8afbc3ac85c3ee8914ee163993f7b7075e43f)
- Release 4.47.1 [`2de160d`](https://github.com/open-wa/wa-automate-nodejs/commit/2de160dd24afe505e16b222855c731fb2a5bd53c)
- ✨ feat: `getFeatures` #2900 [`d89c792`](https://github.com/open-wa/wa-automate-nodejs/commit/d89c792026ee8ae296f52dd38d0ad5d99e7eeae3)
- 🎨 improved incoming chatwoot messages #2907 [`1e3a534`](https://github.com/open-wa/wa-automate-nodejs/commit/1e3a534363af6ffd093e04daea2565f7dca6e342)
- 🔧 fix docs ci [`e01a5e5`](https://github.com/open-wa/wa-automate-nodejs/commit/e01a5e58749c7e1664bc8ade72502a4821c8f0a2)
- 🔧 update gh action [`60cb553`](https://github.com/open-wa/wa-automate-nodejs/commit/60cb553ba13c97e3c31667dab61819afdc88b077)
- 🔧 fix docs ci [`c8c3a0e`](https://github.com/open-wa/wa-automate-nodejs/commit/c8c3a0e639f002087f9a872b86215c56c30809a7)
- 🥚 patches update [`72e3794`](https://github.com/open-wa/wa-automate-nodejs/commit/72e37946212a0f046cf738d566425fe80d8d3b76)
- 🥚 patches update [`ffc9cff`](https://github.com/open-wa/wa-automate-nodejs/commit/ffc9cffbb01ea3ea879007f1788d2b467d382c56)
- 🥚 patches update [`f9e0b0b`](https://github.com/open-wa/wa-automate-nodejs/commit/f9e0b0bd589264e9283416ed3e1b398d8b00c210)
- 🥚 patches update [`08b4904`](https://github.com/open-wa/wa-automate-nodejs/commit/08b4904c22cc2cfffc0e99525771df27703382e8)
- updated types-only package [`aaf4991`](https://github.com/open-wa/wa-automate-nodejs/commit/aaf4991111808a2262a8e4a6819f3941ef12417f)
- 🏷️ fix: export `Call` #2902 [`7e705c4`](https://github.com/open-wa/wa-automate-nodejs/commit/7e705c4e29c17dd6587ba01e8a420cf23847d8d5)
